engine/src/main/java/org/terasology/logic/players/DebugControlSystem.java

```diff
@@ -74,24 +78,45 @@ public void onHideHUD(HideHUDButton event, EntityRef entity) {
         }
     }
 
+    /**
+     * Increases view distance upon receiving an increase view distance event.
+     * @param button The button or key pressed to increase view distance.
+     * @param entity The entity corresponding to event.
```

Comment:
Hooray for adding javadoc! For the `entity` parameter I'm guessing it is the player/client that triggered the event, maybe that would read better in the text? Like "Player entity that triggered the increase" (do confirm that's actually the case - I"m just speculating)